paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt

```diff
@@ -249,15 +234,13 @@ class PageFetcherSnapshotTest {
             )
         )
         advanceUntilIdle()
-        assertThat(fetcherState.newEvents()).isEqualTo(
-            listOf<PageEvent<Int>>(
-                LoadStateUpdate(APPEND, false, Loading),
-                createAppend(
-                    pageOffset = 1,
-                    range = 99..99,
-                    startState = NotLoading.Incomplete,
-                    endState = NotLoading.Complete
-                )
+        assertThat(fetcherState.newEvents()).containsExactly(
+            LoadStateUpdate<PageEvent<Int>>(APPEND, false, Loading),
```

Comment:
Are the generics still needed to be explicitly declared here? If not, can we remove them also while we're doing this cleanup?

Reply:
If I don't provide it, "Not enough information to infer type variable T" error shows up.